index.html

```diff
@@ -228,20 +234,20 @@ <h4> <span class="icon"></span>Skills</h4>
        <section class="contact">
           <h3> I'm always interested in hearing about new projects,
               so if you'd like to chat please get in touch.</h3>
-              <form class="contact-form">
-                 <input type="text" placeholder="Full name" class="input">
-                 <input type="text" placeholder="Email address" class="input">
-                 <textarea  cols="30" rows="10" placeholder="Enter text here"></textarea>
-                 <input type="submit" value="Get in touch" class="btn">
+              <form class="contact-form" action="https://formspree.io/f/meqnpglr" method="post">
+                 <input type="text"  name="full-name" id="full-name" placeholder="Full name" class="input" maxlength="30" required>
+                 <input type="email" name="email" id="email" placeholder="Email address" class="input" required>
+                 <textarea  name="message" id="mess0age" placeholder="Enter text here" maxlength ="500" required></textarea>
+                  <button type="submit" class="submit-btn">Get in touch</button>
```

Comment:
- [ ] Currently, there's a horizontal scroll bar on your page. Kindly fix it.

https://user-images.githubusercontent.com/70483835/165412665-c8fde74d-4896-4f10-98a9-73f8bbc2c6ff.mp4

You can try the following code to fix the issue. I hope It will be solved.
```
    * {
      overflow-x: hidden;
    }
```